opencti-platform/opencti-front/lang/back/en.json

```diff
@@ -206,7 +206,14 @@
   "Display name": "Display name",
   "DLL service": "DLL service",
   "Do you want to delete this support package?": "Do you want to delete this support package?",
+  "Draft change": "Draft change",
   "Draft ids": "Draft ids",
+  "Draft name": "Draft name",
+  "Draft operation": "Draft operation",
+  "Draft patch operation": "Draft patch operation",
+  "Draft patch path": "Draft patch path",
+  "Draft patch value": "Draft patch value",
+  "Draft update patch": "Draft update patch",
```

Comment:
I don't think we need these translations about draft patch, since we don't create the attribute yet